README.md

```diff
@@ -1,9 +1,6 @@
-<p align="center">
-  <img alt="React Native Logo" src="https://reactnative.dev/img/header_logo.svg" width="15%"/>
-</p>
-
 <h1 align="center">
   <a href="https://reactnative.dev/">
+    <img alt="React Native Logo" src="https://reactnative.dev/img/header_logo.svg" width="15%"/><br />
```

Comment:
@raihan71 - Thanks for contributing ! 
I don't know why maintainers have not yet put the logo on README, but I do have some comments. 
- I don't think its important for the logo to act as a link to `https://reactnative.dev/`, so I believe adding it separately above is fine. 
- I won't prefer to nest `<img />` inside of `<h1/>`, though there might be arguments otherwise.

Reply:
Oh yeah thanks for the update, fixed! Hope my first PR is accepted 🙏